src/dodal/devices/watsonmarlow323.py

```diff
@@ -0,0 +1,53 @@
+from enum import Enum
+
+from ophyd_async.core import StandardReadable
+from ophyd_async.epics.signal import epics_signal_r, epics_signal_rw
+
+class WatsonMarlow323Enable(str, Enum):
+     DISABLED = "Disabled"
+     ENABLED = "Enabled"
+
+class WatsonMarlow323Direction(str, Enum):
+     CW = "CW"
+     CCW = "CCW"
+
+class WatsonMarlow323State(str, Enum):
+     STOPPED = "STOP"
+     STARTED = "START"
+
+class WatsonMarlow323(StandardReadable):
```

Comment:
Nit: I would prefer a name that gives some suggestion as to what it does, maybe:

```suggestion
class WatsonMarlow323Pump(StandardReadable):
```

Reply:
Renamed as suggested also updated filenames in 417aaa12897e1aad61c73db30d4f3d6e00974e29.